web/package.json

```diff
@@ -11,6 +11,7 @@
     "url": "https://github.com/electricityMaps/electricitymaps-contrib.git"
   },
   "scripts": {
+    "mockserver": "nodemon ../mockserver/server.js",
```

Comment:
This would require nodemon being installed globally - let's call it `cd ../mockserver && pnpm run start` (but cross-platform friendly?) like before - and maybe convert the mockserver from yarn to pnpm?

Reply:
Hmm I wonder why it worked for me then, I don't have a global version of nodemon installed.
You sure that's needed when it's PNPM doing the command? If it works like NPM it should pick up the local version (could be wrong tho).

Reply:
Weird, it actually works even if I delete `node_modules` in `mockserver` - so don't know where/how it's using `nodemon` 🤔 
I think we should switch from yarn to pnpm in mockserver no matter what though, so we're consistent with package managers :)

Reply:
I think we can keep it as is then and if it ends up breaking I'll fix it at that point.

Reply:
Update to this, Node 19 added a `--watch` flag which can replace nodemon, while we can't and shouldn't update node to v19 now it would solve the issue in the future and allow us to remove nodemon entirely.